lib/simulator_avx.h

```diff
@@ -19,15 +19,15 @@
 
 #include <algorithm>
 #include <cstdint>
-#include <vector>
 
+#include "bits.h"
 #include "statespace_avx.h"
 
 namespace qsim {
 
 /**
- * Quantum circuit simulator with AVX vectorization.
- */
+* Quantum circuit simulator with AVX vectorization.
```

Comment:
Top-level comment: if this is generated code, could you include the script used to generate it in the PR? That would be easier to review, and provides a reference for how these changes were made.

Reply:
I would prefer to not include the script at this point.

Reply:
Discussed offline: submitting the script to the repo is lower priority. #227 has been opened to track that work.